README.md

```diff
@@ -808,6 +809,79 @@ const starkbank = require('starkbank');
 })();
 ```
 
+## Create DynamicBrcodes
+
+You can create simplified dynamic QR Codes to receive money using Pix transactions.
+When a DynamicBrcode is paid, a Deposit is created with the tags parameter containing the character “dynamic-brcode/” followed by the DynamicBrcode’s uuid "dynamic-brcode/{uuid}" for conciliation.
```

Comment:
Mesmo comentário dos outros PR's é "containing the character "dynamic-brcode/"" ou "containing the character "d/"" ?

Reply:
no python tava errado kkk